isis/src/qisis/objs/TargetBody/TargetBody.h

```diff
@@ -186,8 +184,8 @@ namespace Isis {
   //
 
   // operators to read/write TargetBody to/from binary data
-  QDataStream &operator<<(QDataStream &stream, const TargetBody &TargetBody);
-  QDataStream &operator>>(QDataStream &stream, TargetBody &TargetBody);
+  // QDataStream &operator<<(QDataStream &stream, const TargetBody &TargetBody);
+  // QDataStream &operator>>(QDataStream &stream, TargetBody &TargetBody);
```

Comment:
This doesn't cause problems?

Reply:
There's no actual implementation for the operator overloads. I don't think they were hurting anything but I was getting errors that the symbols where missing, which makes sense if there are no functions and are only declared in the header. I will remove them rather than comment them out.